lib/api_stream.go

```diff
@@ -0,0 +1,131 @@
+package sebak
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	common "boscoin.io/sebak/lib/common"
+	"github.com/GianlucaGuarini/go-observable"
+)
+
+type EventStream struct {
+	Events      []string
+	ContentType string
+	observer    *observable.Observable
+	beforeFunc  func()
+	onFunc      OnFunc
+}
+
+type OnFunc func(args ...interface{}) ([]byte, error)
+
+var DefaultContentType = "application/json"
+var OnSerializableFunc = func(args ...interface{}) ([]byte, error) {
+	s, ok := args[1].(common.Serializable)
+	if !ok {
+		return nil, fmt.Errorf("this is not serializable") // TODO(anarcher): Error type
```

Comment:
Could you add a test case for this ?

Reply:
What kind of test case?  I will add more test code. :-)